src/interfaces/SpokePool.ts

```diff
@@ -52,7 +52,7 @@ export enum FillType {
 export interface RelayExecutionEventInfo {
   updatedRecipient: string;
   updatedOutputAmount: BigNumber;
-  updatedMessage?: string;
+  updatedMessage: string;
```

Comment:
Why is this possible now?

Reply:
I'd hit some type errors but I just took a 2nd pass on those and I can work around them OK. https://github.com/across-protocol/sdk/pull/865/commits/3ec724a4d233225ca0b0ddbea93ffd3746c2f9ce